build.sbt

```
@@ -105,7 +105,7 @@ val buildSettings = Seq[Setting[_]](
       ProblemFilters.exclude[MissingClassProblem]("wvlet.airframe.http.internal.*")
     )
   },
-  javacOptions ++= Seq("-source", "11", "-target", "11"),
+  javacOptions ++= Seq("--release", "17"),
```

Still use JDK11 release target until we need to use JDK17-specific features. 
```suggestion
  javacOptions ++= Seq("-source", "11", "-target", "11"),
```